ui/src/app/pages/admin/user/admin-user-bypass.tsx

```diff
@@ -48,6 +48,7 @@ const getBypassedModules = (user: AdminTableUser): Array<AccessModule> => {
     ...(user.rasLinkLoginGovBypassTime
```

Comment:
We can now remove this one, right?  https://github.com/all-of-us/workbench/pull/8372